src/database/repositories/GiveRepository.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-underscore-dangle */
+module.exports = class GiveRepository {
+  constructor(userModal) {
+    this.userModal = userModal;
+  }
+
+  async _give(field, fromID, toID, value) {
+    await this.userModal.updateOne({ id: fromID }, { $inc: { [field]: -value } });
+    await this.userModal.updateOne({ id: toID }, { $inc: { [field]: value } });
+  }
+
+  giveStars(fromID, toID, value) {
+    return this._give('estrelinhas', fromID, toID, value);
+  }
+
+  giveDemons(fromID, toID, value) {
+    return this._give('caçadas', fromID, toID, value);
```

Comment:
This variable is wrong. While not using typescript, it will be hard to know exactly each variable and things, and because we don't have enum, it is more hard. But we are going to ts in a brief, so that's not going to be a problem in the future, but while we dont use it, you can refer to the DatabaseModals to check if everything is okay

https://github.com/ySnoopyDogy/MenheraBot/blob/cef1ce5b93799e74dd16a98eb196cf4443dc87f3/src/structures/DatabaseCollections.js#L73

 in this case, 'caçadas' need to be 'caçados'

Reply:
well seen